Taarafo.Core.Tests.Unit/Services/Foundations/Posts/PostServiceTests.Exceptions.Modify.cs

```diff
@@ -0,0 +1,230 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using Taarafo.Core.Models.Posts;
+using Taarafo.Core.Models.Posts.Exceptions;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.Posts
+{
+    public partial class PostServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowCriticalDependencyExceptionOnModifyIfSqlErrorOccursAndLogItAsync()
+        {
+            // given
+            int minutesInPast = GetRandomNegativeNumber();
```

Comment:
Why do you need minutes in past here? Just throw the Sql Exception out of your datetimebroker?

Reply:
actually, it was one of my questions, that why we need minutes in past? why we use it?

Reply:
@cacti79 when we try to test exceptions - you won't need minutes in the past - you actually won't need minutes at all. just attach the sql exception to the closest dependency.